src/components/Entry.vue

```diff
@@ -143,13 +215,27 @@ export default {
       loadingSpinnerSvg: assets.loadingSpinnerSvg,
       deleteSvg: assets.deleteSvg,
       addSvg: assets.addSvg,
+      extraInputFields: [],
+      dateOfBirth: { month: "", day: "", year: "" },
+      monthList: Array.from({ length: 12 }, (_, i) => i + 1),
+      dayList: Array.from({ length: 31 }, (_, i) => i + 1),
+      yearList: Array.from({ length: 30 }, (_, i) => i + 1989).reverse(),
     };
   },
   created() {
     /** The user type is set as soon as component is created */
     this.userType = this.groupData.userType;
+    this.$store.dispatch("setGroupData", this.groupData);
   },
   computed: {
+    getGroupImages() {
+      return this.groupData.images;
+    },
+    /** Returns if more than one input needs to be validated */
+    isExtraInputValidationRequired() {
+      return this.authType.split(",").length > 1;
+    },
```

Comment:
Along with checking the length, we should also check if the split entities are not empty strings
I know that we would never put such a value in the firestore or DB, but still those kind of cases should be handled gracefully here
So for an auth type like `DOB,` or `,` or `,ID`, this will be wrong because we don't have the complete information

Acha do we have stored anywhere the types of valid auth types?

Reply:
Made the change in the `signup` PR. 

Not really, for now I just made a global variable.